### ISSUE TITLE
suggestions-container add aria-label

### ISSUE THREAD

**Issue by soria2020** [moroshko/react-autosuggest#764]
Hi, 

Is there a way to add aria-label to the suggestion wrapper, this is for accessibility reasons ?
![image](https://user-images.githubusercontent.com/69623832/92382024-32a81800-f10c-11ea-9b09-e636f2f94b2f.png)

**Comment by adamzerella** [comment]
@soria2020 I'm not entirely sure an `aria-label` would make much sense here? AFAIK this attribute would be to describe an element that doesn't have any visible text?
